src/Microsoft.ComponentDetection.Detectors/nuget/FrameworkPackages/FrameworkPackages.net8.0.cs

```diff
@@ -9,18 +9,182 @@ internal partial class FrameworkPackages
 {
     internal static class NETCoreApp80
     {
-        internal static FrameworkPackages Instance { get; } = new(NuGetFramework.Parse("net8.0"), NETCoreApp70.Instance)
+        internal static FrameworkPackages Instance { get; } = new(NuGetFramework.Parse("net8.0"), "Microsoft.NETCore.App", NETCoreApp70.Instance)
```

Comment:
Let's create a shared static constant for these values: "Microsoft.NETCore.App", "Microsoft.AspNetCore.App", and "Microsoft.WindowsDesktop.App" that we can consume across all files.